clients/unshaded/src/main/java/tachyon/client/file/BaseFileSystem.java

```diff
@@ -49,12 +47,14 @@
 */
 @PublicApi
 public class BaseFileSystem implements FileSystem {
+  private FileSystemContext mContext;
```

Comment:
Since its of type `FileSystem`, do you think its still better to full specify the context type? It will only be used in this class.